src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -365,74 +396,93 @@ export default class PhotoField extends React.Component {
     // at this point, the onZoom event has not resized the canvas-
     // this forces the canvas back into move bounds if the zoom pulls a canvas edge into the cropBox
     //   after the canvas has rendered with new width values
-    window.requestAnimationFrame(() => this.maybeMoveCanvasWithinBounds({}, zoomWarn));
+    window.requestAnimationFrame(() => this.moveCanvasWithinBounds({}, zoomWarn));
   }
 
   setCropBox = () => {
     window.requestAnimationFrame(() => {
-      // center the cropbox using container width
-      const containerWidth = this.refs.cropper.getContainerData().width;
-      // const containerWidth = this.refs.container.offsetWidth;
-      const smallScreen = isSmallScreen(this.state.windowWidth) || onReviewPage(this.props.formContext.pageTitle);
-      const left = smallScreen ? (containerWidth / 2) - 120 : (containerWidth / 2) - 150;
-      this.refs.cropper.setCropBoxData({
-        top: 0,
-        left,
-        height: smallScreen ? SMALL_CROP_BOX_SIZE : LARGE_CROP_BOX_SIZE,
-        width: smallScreen ? SMALL_CROP_BOX_SIZE : LARGE_CROP_BOX_SIZE
-      });
+      // sometimes this callback is called before the new cropper has mounted
+      // check if the cropper instance has canvasData before setting its values
+      if (!_.isEmpty(this.refs.cropper.getCanvasData())) {
```

Comment:
I noticed this was causing an error in the console- nothing visible. Sometimes this callback is executed while the component is unmounting and these values aren't ready. Since `setCropBox` is called again once the component is initialized, can safely by pass this when the data isn't available.